src/Controller/GameController.php

```diff
@@ -24,17 +24,28 @@ class GameController extends AbstractController
     /**
      * Show first level
      */
-    public function index(?string $action): string
+    public function index(?string $action, ?string $id): string
```

Comment:
why not ?int for $id? if you convert it now, you won't have to do it with a filter_var later

Reply:
you may also give a default value: int $id = 1

Reply:
done